server/src/main/java/org/opensearch/Application.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch;
+
+import org.opensearch.identity.Subject;
+
+/**
+ * A service that transmits data to and/or from OpenSearch
+ *
+ * @opensearch.experimental
+ */
+public interface Application extends Subject {}
```

Comment:
What do you think about removing Subject and just adding 'getPrinciple()` for this interface.  I don't think it makes sense for a PluginInfo to have an authenticate(...) method

Reply:
Sounds good!